### PR TITLE
Makes the gibber give skins when gibbing humans. Also adds craftable human suits from human skin.

### PR DESCRIPTION
**Pull request by Firecage**
Basically everything it says in the OP. When you put someone into the gibber, it now also spits out a skin/hide. For most species it will only give a generic hide since I cannot into spriting. For humans and lizards, it gives human/lizard skins/hides respectively.

5 human skins can be used to craft a bloated human suit, which looks like the changeling spacesuit.

Why not? This is creepy.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Spooky

**Comment by neersighted** [comment]
This is horrible :+1:

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -250,6 +250,23 @@
 	//min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	flags = NODROP
 
+/obj/item/clothing/suit/hooded/bloated_human	//OH MY GOD WHAT HAVE YOU DONE!?!?!?
+	name = "bloated human suit"
```
Comment (Dorsisdwarf) [inline code/modules/clothing/suits/miscellaneous.dm]: totally make its name the same as the lings' (bloated flesh mass right?) for maximum honks as clown.

Comment (Firecage) [inline code/modules/clothing/suits/miscellaneous.dm]: Nah Dorsi, thats just silly.

**Comment by KazeEspada** [comment]
Yesss, now i can dressss asss my fursssona.

**Comment by Niknakflak** [comment]
We can finally have lizards who larp as humans.

**Comment by Malkevin** [comment]
Good bye horses....

File: code/game/objects/items/stacks/sheets/leather.dm [hunk 2]
```diff
@@ -5,6 +5,21 @@
 	icon_state = "sheet-hide"
 	origin_tech = null
 
+var/global/list/datum/stack_recipe/human_recipes = list( \
+	new/datum/stack_recipe("bloated human costume", /obj/item/clothing/suit/hooded/bloated_human, 5, on_floor = 1), \
+	)
```
Comment (duncathan) [inline code/game/objects/items/stacks/sheets/leather.dm]: I hate this but it's not your fault

File: code/modules/clothing/suits/miscellaneous.dm [hunk 3]
```diff
@@ -249,6 +249,23 @@
 	//min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
 	flags = NODROP
 
+/obj/item/clothing/suit/hooded/bloated_human	//OH MY GOD WHAT HAVE YOU DONE!?!?!?
+	name = "bloated human suit"
+	desc = "A horribly bloated suit made from human skins."
+	icon_state = "lingspacesuit"
+	item_state = "labcoat"
+	body_parts_covered = CHEST|GROIN|ARMS
+	allowed = list(,)
```
Comment (duncathan) [inline code/modules/clothing/suits/miscellaneous.dm]: Why the comma? Is the list expected to have two elements?

**Pull request merged**